examples/myapp-03/myapp-ebpf/src/main.rs

```diff
@@ -68,18 +68,18 @@ fn try_xdp_firewall(ctx: XdpContext) -> Result<u32, ()> {
     if h_proto != ETH_P_IP {
         return Ok(xdp_action::XDP_PASS);
     }
-    let source = u32::from_be(unsafe { *ptr_at(&ctx, ETH_HDR_LEN + offset_of!(iphdr, saddr))? });
+    let daddr = u32::from_be(unsafe { *ptr_at(&ctx, ETH_HDR_LEN + offset_of!(iphdr, daddr))? });
```

Comment:
changing from source to destinateion address will require updating chapter 2 too.

Reply:
not just the code, but the prose and log output too.